### PR TITLE
Make it easier to follow release process steps

### PR DESCRIPTION
**Pull request by akshay-ap**
## Description
Modify `RELEASE_PROCESS.md` file and make it easy to follow steps.

## Is this PR related with an open issue?

Related to Issue #521 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] Follows the code style of this project.
- [x] Tests Cover Changes
- [x] Documentation

### CODE REVIEW

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -2,18 +2,32 @@
...
+
 - Push the feature branch to GitHub.
+
+  `git push origin feature/bumpversion-to-v0.2.5`
+
+## Step 2: Merge changes to develop branch
+
 - Make a pull request from the just-pushed branch.
 - Wait for all the tests to pass!
 - Merge the pull request into the `develop` branch.
```
Comment (alexcos20) [inline RELEASE_PROCESS.md:27]: maybe "main" branch ?

File: RELEASE_PROCESS.md [hunk 2]
```diff
@@ -22,6 +36,9 @@ SPDX-License-Identifier: Apache-2.0
   - For the target, select the `develop` branch, or the just-merged commit.
```
Comment (alexcos20) [inline RELEASE_PROCESS.md:36]: "main" branch ?

**Review by alexcos20 — CHANGES_REQUESTED** [review]

File: RELEASE_PROCESS.md (hunk 1 above)
Comment (calina-c) [reply RELEASE_PROCESS.md:27]: I agree. It should be `main`. CC: @akshay-ap

**Review by calina-c — APPROVED** [review]

**Comment by akshay-ap** [comment]
@calina-c Thanks for quick fix!

**Pull request merged**